### PR TITLE
Added a Spotify clone

### PR DESCRIPTION
**Pull request by Anjali-ui28**
# Zero To Mastery - Webblocks 2022 📦📦

## PR contains:
I have made a Spotify clone which has a similar UI as of Spotify web  . It also contains songs which can be played by the user .It contains play/pause, forward/backward buttons to make it more useful.

## Breaking changes
None

## Screenshots
Screenshot 2022-10-15 060930

### CODE REVIEW

File: SpotifyClone/script.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+console.log("Welcome to Spotify");
```
Comment (sophiabrandt) [inline SpotifyClone/script.js:1]: Please remove `console.log` from code that you ship to production.

File: SpotifyClone/script.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+songItem.forEach((element, i) => {
+    console.log(element, i);
+    element.getElementsByTagName("img")[0].src = songs[i].coverPath;
+    element.getElementsByClassName("songName")[0].innerText = songs[i].songName;
+
+
+})
+
+
+//audioElement.play();
```
Comment (sophiabrandt) [inline SpotifyClone/script.js:26]: Please remove outdated code.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]
Please read the guide on how to contribute to this repository.

Please check if you followed it correctly.

From a first glance, you need to implement the following changes:
a) the contribution is in an incorrect folder
b) `meta.json` with values is missing
c) you use artwork and music files that are licensed or you didn't provide proof that you own the rights or that these works can be used under a permissible license.

**Comment by MattCSmith** [comment]
Closing due to unresolved change request. Feel free to resolve and reopen.

**Pull request closed without merging**